daffodil-runtime1/src/main/scala/org/apache/daffodil/debugger/InteractiveDebugger.scala

```diff
@@ -1231,21 +1231,6 @@ class InteractiveDebugger(runner: InteractiveDebuggerRunner, eCompilers: Express
         }
       }
 
-      object InfoOccursIndex extends DebugCommand with DebugCommandValidateZeroArgs {
```

Comment:
While we're reording things, it might be worth it to reorder the ``info`` subcommands Seq so they're alphabetical, and to make it so it's not on one ginormous line.